scripts/ts_indices/extract_ts_indices.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+import os
+import xarray as xr
+from calendar import monthrange
+from mpi4py import MPI
+import time
+
+references = 'Copernicus Climate Change Service (C3S) (2017): ERA5: Fifth generation of ECMWF atmospheric reanalyses of the global climate'
+
+licence = """
```

Comment:
In future we should put the C3S licence in a separate file and read it in when assigning the attribute. Makes this code a bit cleaner